### DISCUSSION TITLE
projected-unfold usage

### DISCUSSION THREAD

**Discussion by xinting1118** [band-unfolding/bandup#39] [Q&A] [answered]
Hello,
I reported an error when using the projected-unfold, FileNotFoundError: No such file or directory: ‘~/orbital_projections/KptsInfo_iSpin_1.pkl.’ How to solve it?

**Comment by stepan-tsirkin** [comment, marked as answer]
Could you post the full stack trace of errors, and the command that you are executing

**Reply by xinting1118** [reply]
Hello, 
    Thank you very much for your reply. Now that the issue has been fixed, I was using Bandup project-unfold without running the --orbital command first which caused the error.
    But I had a problem running Bandup projection-unfold -atoms -orbs, and the atomic and orbital projected bands of the primitive cell did not correspond to the result of the unfolding bands of the supercell. I'd be grateful if you could answer that.
```
CONTCAR
   1.00000000000000
     6.3242001533999996    0.0000000000000000    0.0000000000000000
    -3.1621000766999998    5.4769179913999997    0.0000000000000000
     0.0000000000000000    0.0000000000000000   30.4978008269999989
   Mo   S
     4     8
Direct
  0.3333300050000005  0.1666699949999995  0.5000000000000000
  0.3333300050000005  0.6666700239999983  0.5000000000000000
  0.8333299760000017  0.1666699949999995  0.5000000000000000
  0.8333299760000017  0.6666700239999983  0.5000000000000000
  0.0000000000000000  0.0000000000000000  0.4485599989999969
  0.0000000000000000  0.5000000000000000  0.4485599989999969
  0.5000000000000000  0.0000000000000000  0.4485599989999969
  0.5000000000000000  0.5000000000000000  0.4485599989999969
  0.0000000000000000  0.0000000000000000  0.5514400010000031
  0.0000000000000000  0.5000000000000000  0.5514400010000031
  0.5000000000000000  0.0000000000000000  0.5514400010000031
  0.5000000000000000  0.5000000000000000  0.5514400010000031
```
I use this command bandup projected-unfold -at 1 -orbs dz2 to output the dz2 orbital of the 1st Mo projected-unfold band.

**Reply by stepan-tsirkin** [reply]
It is hard to tell what is wrong without seeing what you get and how it is wrong

**Reply by xinting1118** [reply]
Hello,
Thank you very much for your reply. 
The first picture is obtained by running the bandup projected-unfold -at 1-4 command, which is used to calculate the band unfolding of the Mo element projection.
![unfolded_EBS_orb_dependent_Mo](https://github.com/band-unfolding/bandup/assets/127070495/7f968646-d1b0-4fee-869f-f2d812d785d3)
The second picture is obtained by running the bandup projected-unfold -at 5-12 command, which is used to calculate the band unfolding of the S element projection.
![unfolded_EBS_orb_dependent_S](https://github.com/band-unfolding/bandup/assets/127070495/f339f94b-ef35-4687-978b-6824c7db7785)
The image below shows the projected bands of the Mo and S elements of the MoS2 primitive cell.
![projected_band](https://github.com/band-unfolding/bandup/assets/127070495/75c69ebe-48df-4031-9c63-9fb73165a06a)
The data processed by Bandup has less Mo weight and more S weight, which is opposite to the projected band of the primitive cell.
I'd appreciate it if you could answer me.
